### ISSUE TITLE
Latest version (2024.04.0+735) shows package functions in reverse alphabetical order

### ISSUE THREAD

**Issue by jzadra** [rstudio/rstudio#14677]
![image](https://github.com/rstudio/rstudio/assets/25590353/1019baf5-03fc-4769-b00d-d071d98d73a3)

**Comment by ronblum** [comment]
@jzadra Thank you for raising this! Do you have GitHub Copilot enabled? The order will get reversed to accommodate code suggestions (although in the Editor Pane). Also, can you please provide your system info?

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       :

**Comment by ronblum** [comment]
@jzadra So I think what we're looking for here is that the behavior in the Console Pane should be the same, whether or not Copilot is enabled? That is, it should always look like this in the Console Pane:

![autocomplete_order](https://github.com/rstudio/rstudio/assets/31009952/4590d83a-6f13-44bb-9019-7705c1cc01fb)

**Comment by jzadra** [comment]
Correct.  And to clarify, when I'm in the editor/source pane, the behavior is the same (reverse order).  But I take it you are saying that since there are no autocompletions from copilot in the console, there is no need for it to be reverse order?

I think there may be something to be said for maintaining consistency in this menu between source/console, meaning if it has to be reversed in source, perhaps it be the same in console.  But that's probably open for debate.
